src/components/ShowList.jsx

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
 import { useCollection } from 'react-firebase-hooks/firestore';
-import firebase from 'firebase/app';
-import 'firebase/firestore';
-import '../lib/firebase';
+import db from '../lib/firebase';
+import Navigation from './Navigation';
 
 function ShowList() {
   const [value, loading, error] = useCollection(
-    firebase.firestore().collection('items').orderBy('date'),
+    db.collection('items').orderBy('date'),
```

Comment:
Was this part pulled from the other pair's branch? 

Make sure to coordinate with them because they made some changes here to properly render items associated with the user token.